client/src/App.js

```diff
@@ -1,5 +1,11 @@
+import Ohm from './pages/Ohm';
+
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Ohm />
```

Comment:
I think this means "ohm" what monks say? like the meditation "ohm"? Maybe you could rename this ?!

Reply:
Thank you for the review!
You are right, it means "ohm", I wanted the site to start with a short kind of meditation. Naming is not final yet;-)